pkg/k8s/k8s_service.go

```diff
@@ -126,8 +126,8 @@ func buildPorts(tcpPorts, udpPorts []int) []v1.ServicePort {
 }
 
 // prepareService constructs a new Service object with the specified parameters.
-func prepareService(namespace, name string, labels, selectorMap map[string]string,
-	tcpPorts, udpPorts []int) (*v1.Service, error) {
+func prepareService(namespace, name string, labels, annotations, selectorMap map[string]string,
```

Comment:
same to this one, using the previous style

Reply:
https://github.com/celestiaorg/knuu/pull/152/commits/e371faab7c9f662bef4d180d9ad496c4a8a6a60f